policyengine_us/variables/gov/states/hi/tax/income/deductions/itemized/hi_itemized_deduction.py

```diff
@@ -0,0 +1,31 @@
+from policyengine_us.model_api import *
+
+
+class hi_itemized_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Hawaii itemized deduction"
+    unit = USD
+    documentation = (
+        "https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=15"
+        "https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=32"  # total itemized deduction worksheet
+    )
+    definition_period = YEAR
```

Comment:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/4f2bd316-a938-4ca5-a7d8-a673911c7002)

have we included this condition?